styles.css

```diff
@@ -71,8 +78,201 @@ a {
   font-size: 15px;
 }
 
+.connect-links {
+  display: flex;
+  flex-direction: row;
+  padding-left: 0;
+}
+
 .social-icons {
   height: 22px;
   width: 22px;
   padding-right: 15px;
 }
+
+#projects {
+  background-color: #f7f7f9;
+  padding-top: 100px;
+  padding-bottom: 30px;
+  display: flex;
+  flex-direction: column;
+}
```

Comment:
- [ ] As indicated by the previous reviewer and stated as part of the project requirement, you should use Grid layout for the projects container and FlexBoxy to layout elements of a project card.
![image](https://user-images.githubusercontent.com/44057863/170841685-c04c09cf-213c-476f-b023-02c75c78e6dd.png)